README.md

```diff
@@ -149,7 +188,25 @@ The ancestors have provided an example of one route; you do the other six!
 GET '/warriors/:id'
   * This is the show route, which finds a warrior by ID, and displays information about that warrior.
 
-Replace this with your answer
+GET '/warriors'
+  *Displays information about all the warriors
+
+GET 'warriors/new'
+  *Returns a form to create a new warrior
+
+POST '/warriors'
+  * Creates a new warrior
+
+PUT '/warriors/:id'
+  * Finds a warrior and rewrites all of the warrior's information
+
+GET '/warriors/:id/edit'
+  * Finds a warrior and rewrites part or all of the warrior's information
```

Comment:
The edit view simply displays information about the warrior, gathering the requested changes from the user.  The update route actually updates (or rewrites) the warrior's information.